extensions/wikia/Search/classes/Test/Controller/SearchControllerTest.php

```diff
@@ -570,150 +443,83 @@ public function testPaginationWithConfigNoResults2() {
 	 * @covers WikiaSearchController::pagination
 	 */
 	public function testPaginationWithConfig() {
-		$mockController		=	$this->searchController->setMethods( array( 'getVal', 'setVal' ) )->getMock();
-		$mockTitle			=	$this->getMockBuilder( 'Title' )->disableOriginalConstructor()->getMock();
-		$mockResponse		=	$this->getMock( 'WikiaResponse', array( 'redirect', 'setVal' ), array( 'html' ) );
-		$mockRequest		=	$this->getMock( 'WikiaRequest', array( 'getVal' ), array( array() ) );
-		$configMethods		=	array( 'getResultsFound', 'getPage', 'getQuery', 'getNumPages', 'getInterWiki',
-										'getSkipCache', 'getDebug', 'getNamespaces', 'getAdvanced',
-										'getLimit', 'getPublicFilterKeys', 'getRank' );
-		$mockConfig			=	$this->getMock( 'Wikia\Search\Config', $configMethods );
-		$mockQuery = $this->getMock( 'Wikia\Search\Query\Select', array( 'getSanitizedQuery' ), array( 'foo' ) );
-
-		$mockWgRefl = new ReflectionProperty( 'WikiaSearchController', 'wg' );
-		$mockWgRefl->setAccessible( true );
-		$mockWgRefl->setValue( $mockController, (object) array( 'Title' => $mockTitle ) );
-
-		$this->getStaticMethodMock( 'SpecialPage', 'getTitleFor' )->expects( $this->any() )
-				->method( 'getTitleFor' )
-				->will( $this->returnValue( $mockTitle ) );
+		$mockQuery = $this->getMock( Wikia\Search\Query\Select::class, [ 'getSanitizedQuery' ], [ 'foo' ] );
+		$mockQuery
+			->expects( $this->once() )
+			->method( 'getSanitizedQuery' )
+			->will( $this->returnValue( 'foo' ) );
 
-		$mockController
-			->expects	( $this->at( 0 ) )
-			->method	( 'getVal' )
-			->with		( 'config', false )
-			->will		( $this->returnValue( $mockConfig ) )
-		;
+		$configMethods = [ 'getResultsFound', 'getPage', 'getQuery', 'getNumPages', 'getInterWiki',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getInterWiki" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)